runtime/common/src/lib.rs

```diff
@@ -75,7 +75,7 @@ const_assert!(NORMAL_DISPATCH_RATIO.deconstruct() >= AVERAGE_ON_INITIALIZE_RATIO
 
 // Common constants used in all runtimes.
 parameter_types! {
-	pub const BlockHashCount: BlockNumber = 2400;
+	pub const BlockHashCount: BlockNumber = 4096;
```

Comment:
I suggested 4096 because transaction mortality is a power of 2. Yes, you can set it higher in the transaction (like setting it to 4096 in order to get 2400), but it led to confusion from people about they their transaction was only valid for 2400 when they set mortality to 4096. This gives the extra time without needing to explain to people why their 8192 tx is only valid for 4200.

Reply:
Ah yeah good point. Quick follow up, would increasing this a little bit more push this over into the 4th tier of the trie storage data structure used to hold the mapping data?

Reply:
And yes to Joe's point, it took me some time to dig through the code to understand fully what was going on with the mortality, power of 2, and block hash count relationship.

Reply:
Okay fine :)